Gemfile

```diff
@@ -53,7 +53,7 @@ gem 'listen', '3.0.8'
 #
 # From a location on your disk:
 #
-gem 'aweplug', git: 'https://github.com/awestruct/aweplug'
+gem 'aweplug', git: 'https://github.com/Dantheman720/aweplug', :branch => 'DEVELOPER-3647'
```

Comment:
I submitted the PR. I'll resubmit this PR once Aweplug is merged. This was mainly for testing purposes.